modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,127 @@
+
+[IMPORTANT]
+====
+You are responsible for arranging access to {oracle} base media repositories and {oracle} installation media.
+====
+
+ifeval::[{suma-content} == true]
+
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+== Server Requirements
+
+Before you begin, check that your {productname} Server meets the requirements at xref:installation:hardware-requirements.adoc[].
+
+Taskomatic uses one CPU core, and requires at least 3072{nbsp}MB of RAM.
+To ensure that taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+
+----
+taskomatic.java.maxmemory=3072
+----
+
+Restart Taskomatic:
+----
+systemctl restart taskomatic
+----
+
+
+
+== Channel and Repository Management
+
+
+
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
+* For {oracle} 7:
++
+From the {webui}, add [systemitem]``Oracle Linux 7 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux7-x86_64``.
++
+* For {oracle} 8:
++
+From the {webui}, add [systemitem]``Oracle Linux 8 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux8-x86_64``.
+. Synchronize the {productname} Server with the {SCC}.
+You can do this using the {webui}, or by running [command]``mgr-sync`` at the command prompt.
+. Add the new channel to your activation key.
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+[IMPORTANT]
+====
+For {oracle} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
+You will require packages from both channels.
+If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+====
+
+
+.Procedure: Adding Client Tools Channels
+. At the command prompt on the {productname} Server, as root, install the [package]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the {oracle} base, updates, and client channels, specifying the {oracle} version and architecture:
++
+----
+spacewalk-common-channels -a x86_64 oracle8 \
+oracle8-uyuni-client oracle8-appstream
+----
+
+
+[NOTE]
+====
+The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
+====
+
+endif::[]
+
+
+There are two ways to check if a channel has finished synchronizing:
+
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
++
+This dialog displays a completion bar for each product when they are being synchronized.
+* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
```

Comment:
```suggestion
* Check the synchronization log file at the command prompt:
+
----
tail -f /var/log/rhn/reposync/channel-label.log
----
```
Additionally you probably must indicate that "channel-label" is a replaceable.  Maybe, the following sentence is good enough.

Reply:
This is in every client chapter, it will need to be done everywhere.

Reply:
Done in PR#266